JASP-Engine/JASP/po/R-nl.po

```diff
@@ -7264,6 +7394,35 @@ msgstr ""
 msgid "Path Diagrams"
 msgstr "Paddiagrammen"
 
+msgid ""
+"- *Bayes factor*: If one-sided test is requested:\n"
+"- BF+0: Bayes factor that quantifies evidence for the one-sided alternative "
+"hypothesis that the population mean is larger than the test value\n"
+"- BF-0: Bayes factor that quantifies evidence for the one-sided alternative "
+"hypothesis that the population mean is smaller than the test value\n"
+"- BF0+: Bayes factor that quantifies evidence for the null hypothesis "
+"relative to the one-sided alternative hypothesis that the population mean is "
+"larger than the test value\n"
+"- BF0-: Bayes factor that quantifies evidence for the null hypothesis "
+"relative to the one-sided alternative hypothesis that that the population "
+"mean is smaller than the test value\n"
+"- **p**: p-value corresponding to t-statistic."
+msgstr ""
+"- *Bayes factor*: Als een eenzijdig toets wordt gevraagd:\n"
+"- BF+0: Bayes factor die bewijs quantificeert voor de eenzijdige "
+"alternatieve hypothese dat het populatie gemiddelde groter is dan de "
```

Comment:
Is "populatiegemiddelde" niet een woord?

Reply:
T ziet er wel een beetje lomp uit I guess

Reply:
OK